mappings/net/minecraft/data/server/RecipesProvider.mapping

```diff
@@ -232,3 +232,11 @@ CLASS net/minecraft/class_2446 net/minecraft/data/server/RecipesProvider
 		ARG 2 output
 		ARG 3 experience
 		ARG 4 cookingTime
+	METHOD method_36325 offerReversibleCompactingRecipes (Ljava/util/function/Consumer;Lnet/minecraft/class_1935;Lnet/minecraft/class_1935;)V
```

Comment:
I would call those 'small compacting' recipes, since I believe only a few recipes like copper and quartz use them. For now, only raw metal has a specific function for reversible compacting recipes.